### PR TITLE
feat: enable gpu snarks by default

### PR DESCRIPTION
**Pull request by dignifiedquire**
They work well enough that it makes sense to run with them when available

@laser for awareness

### CODE REVIEW

File: fil-proofs-tooling/Cargo.toml [hunk 1]
```diff
@@ -42,5 +42,5 @@ log = "0.4.8"
 uom = "0.25.0"
 
 [features]
-default = []
+default = ["gpu"]
```
Comment (cryptonemo) [inline fil-proofs-tooling/Cargo.toml]: By making this the default, does it mean a GPU is required? Or that it'll be used by default if detected?

Comment (dignifiedquire) [reply fil-proofs-tooling/Cargo.toml]: it means it well run the detection and use it if it finds a compatible one

**Comment by dignifiedquire** [comment]
Blocked by https://github.com/cogciprocate/ocl/pull/163

**Comment by dignifiedquire** [comment]
> Blocked by [cogciprocate/ocl#163](https://github.com/cogciprocate/ocl/pull/163)

actually it is not, seems some other issue

**Pull request merged**